rts/src/eta/runtime/apply/Apply.java

```diff
@@ -6,208 +6,229 @@
-                fun.apply(context, p1, p2, p3, Void._void);
-            }
-        };
-
-    public static RtsFun ap_pppvo_fast = new RtsFun() {
-            @Override
-            public void enter(StgContext context) {
-                StgClosure fun = context.R(1);
-                StgClosure p1 = context.R(2);
-                StgClosure p2 = context.R(3);
-                StgClosure p3 = context.R(4);
-                Object o = context.O(1);
-                fun.apply(context, p1, p2, p3, null, o);
-            }
-        };
-
-    public static RtsFun ap_pppp_fast = new RtsFun() {
-            @Override
-            public void enter(StgContext context) {
-                StgClosure fun = context.R(1);
-                StgClosure p1 = context.R(2);
-                StgClosure p2 = context.R(3);
-                StgClosure p3 = context.R(4);
-                StgClosure p4 = context.R(5);
-                fun.apply(context, p1, p2, p3, p4);
-            }
-        };
-
-    public static RtsFun ap_ppppp_fast = new RtsFun() {
-            @Override
-            public void enter(StgContext context) {
-                StgClosure fun = context.R(1);
-                StgClosure p1 = context.R(2);
-                StgClosure p2 = context.R(3);
-                StgClosure p3 = context.R(4);
-                StgClosure p4 = context.R(5);
-                StgClosure p5 = context.R(6);
-                fun.apply(context, p1, p2, p3, p4, p5);
-            }
-        };
-
-    public static RtsFun ap_pppppp_fast = new RtsFun() {
-            @Override
-            public void enter(StgContext context) {
-                /* TODO: Verify implementation */
-                StgClosure fun = context.R(1);
-                StgClosure p1 = context.R(2);
-                StgClosure p2 = context.R(3);
-                StgClosure p3 = context.R(4);
-                StgClosure p4 = context.R(5);
-                StgClosure p5 = context.R(6);
-                StgClosure p6 = context.R(7);
-                fun.apply(context, p1, p2, p3, p4, p5, p6);
-            }
-        };
-
-    public static RtsFun PAP_apply = new RtsFun() {
-            @Override
-            public void enter(StgContext context) {
-                StgPAP pap = (StgPAP) context.R(1);
-                AbstractArgumentStack stack = (AbstractArgumentStack) context.O(1);
-                context.merge(stack);
-                StgClosure fun = pap.fun;
-                fun.enter(context);
-            }
-        };
+    public static RtsFun ap_0_fast = new Ap0Fast();
+    public static RtsFun ap_v_fast = new ApVFast();
+    public static RtsFun ap_n_fast = new ApNFast();
+    public static RtsFun ap_l_fast = new Ap1Fast();
+    public static RtsFun ap_f_fast = new ApFFast();
+    public static RtsFun ap_d_fast = new ApDFast();
+    public static RtsFun ap_o_fast = new ApOFast();
+    public static RtsFun ap_p_fast = new ApPFast();
+    public static RtsFun ap_pv_fast = new ApPvFast();
```

Comment:
Should be `ApPVFast`, make sure you capitalise all the letters after `Ap` and before `Fast`. Do the same for all the remaining `Ap*Fast`